source/_integrations/escea.markdown

```diff
@@ -0,0 +1,47 @@
+---
+title: Escea
+description: Instructions on how to integrate Escea fireplaces with Home Assistant.
+ha_category:
+  - Climate
+ha_release: 2022.9
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_codeowners:
+  - '@lazdavila'
+ha_domain: escea
+ha_homekit: true
+ha_platforms:
+  - climate
+---
+
+The Escea integration allows monitoring and control of local [Escea](https://escea.com/) fireplaces. These fireplaces are manufactured in New Zealand and sold throughout Australia and New Zealand.
+
+## Supported hardware
+
+Support all current Escea fireplaces with the Smart Heat feature, including the DF, DL, DS, and DX series.
+Though these fireplaces are supported, they can only be integrated if they were connected via Ethernet when installed (i.e., these fireplaces do not support Wifi).
+
+{% include integrations/config_flow.md %}
+
+## Network settings
+
+The Escea system uses UDP over the local network to find and communicate with Escea devices. For this to work properly, UDP port 3300 must be available for broadcasting and messaging fireplaces, and similarly, UDP port 3300 must be available to listen for responses. The integration currently listens on `0.0.0.0` and broadcasts to all broadcast IPv4 local addresses, which is not configurable.
```

Comment:
Shouldn't it be listening to the interfaces configured in the Home Assistant network manager? Claiming a port across all interfaces (including the internal Docker interface), seems kinda unneeded and makes it really easy to run into port collisions (that can't be avoided as far as I can see now).

Reply:
Hi, sorry, I guess you are right. The fireplace is hard coded to receive the UDP requests on port 3300, and hard coded to respond via UDP messages on port 3000. If I understand right, instead of waiting on a response on 0.0.0.0, it should be on a more specific IP provided by Network Manager? I don't know how to arrange that... if you have some similar code I could look at, that would really help? Regards, Laz.

Reply:
The documentation can be found here: <https://developers.home-assistant.io/docs/network_discovery?_highlight=network#network>

That will help you get the interfaces active, selected, and available to listen on.

Reply:
Thanks - I will look into that!

Reply:
I'll go ahead and merge this one for now, to make the documentation in the `next` branch to match the existing code 👍 

Please open up a new docs PR in case this changes.

../Frenck